packages/esm-providers-app/src/table/provider-details.component.tsx

```diff
@@ -21,10 +21,10 @@ const ProviderDetails: React.FC<ProviderDetailsProps> = ({ providerUuid }) => {
   const licenseAttr = provider?.attributes?.find((attr) => attr.attributeType.display === 'Practising License Number');
   const nationalID = provider?.attributes?.find((attr) => attr.attributeType.display === 'Provider National Id Number');
   const dateAttr = provider?.attributes?.find((attr) => attr.attributeType.display === 'License Expiry Date');
-  const phoneNumber = provider?.attributes?.find((attr) => attr.attributeType.display === 'Phone number');
-  const qualification = provider?.attributes?.find((attr) => attr.attributeType.display === 'Provider qualification');
+  const phoneNumber = provider?.attributes?.find((attr) => attr.attributeType.display === 'Provider Telephone');
```

Comment:
```suggestion
  const attributeMap = {
  licenseAttr: 'Practising License Number',
  nationalID: 'Provider National Id Number',
  dateAttr: 'License Expiry Date',
  phoneNumber: 'Provider Telephone',
  qualification: 'Provider Qualification',
  registrationNumber: 'License Body',
  emailAddress: 'Provider Address'
};

const attributes = Object.entries(attributeMap).reduce((acc, [key, display]) => ({
  ...acc,
  [key]: provider?.attributes?.find(attr => attr.attributeType.display === display)
}), {});
```

we can retrieve the attributes  e.g
```tsx
const { licenseAttr, nationalID, dateAttr, phoneNumber, qualification, registrationNumber, emailAddress } = attributes;
```

Reply:
updated the retrieve of the attributes